docs/source/dev_guide/algorithms/k_nearest_neighbors/k-nearest-neighbors-knn-classifier.rst

```diff
@@ -68,9 +76,19 @@ Feature vectors from any bucket belong to the region of the space
 defined by tree nodes on the path from the root node to the
 respective leaf.
 
+Brute Force
+-----------
+
+The main idea of the Brute Force method is to calculate all of the distances from each query feature vector
+to each reference feature vector in the training data set and then select :math:`k` objects 
+that are closest to the query point.
```

Comment:
what do you mean by "not calculated inside"?

Reply:
We don't calculate distances in the implementation, while I see "to calculate all of the distances" in the doc. I confuses, because calculation of distances is really bad idea (but possbile) to do in the brute-force kNN.

To preserve consistency and common logic, I expect to to here explanation of the brute-force approach of kNN with reference to bibliography that can be used for further reading.

Reply:
@amgrigoriev  need your help here

Reply:
My suggestion: "Brute Force method calculates all of the squared distances from each query feature vector
to each reference feature vector in the training data set and then for every query feature vector selects :math:`k` objects from the training set that are closest to it."
In reality,  squared distances are incomplete ones but details can't be explained in a couple of sentences. It would be better to skip internal details in a brief description.